tests/getenclave_nondebugenc/enc/enc.c

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/enclave.h>
+#include <openenclave/internal/tests.h>
+#include "getenclave_t.h"
+
+oe_result_t test_get_enclave_ecall(oe_enclave_t* enclave_param)
+{
+    oe_result_t result = OE_UNEXPECTED;
+    oe_enclave_t* enclave;
+
+    if (!(enclave = oe_get_enclave()))
+        goto done;
+
+    if (enclave_param != enclave)
+        goto done;
+
+    if (test_get_enclave_ocall(enclave) != OE_OK)
+        goto done;
+
+    result = OE_OK;
+
+done:
+    return result;
+}
+
+#if defined(__GNUC__)
+__attribute__((constructor)) void global_constructor()
+{
+    OE_TEST(oe_get_enclave() != NULL);
+}
+#endif
+
+OE_SET_ENCLAVE_SGX(
+    0,    /* ProductID */
+    0,    /* SecurityVersion */
+    false, /* NotAllowDebug */
```

Comment:
This seems like the same code as the other test. Could you just make this a build-time define and compile the enclave twice in the other test?

I'd like to avoid duplicating the code if possible.

Reply:
The projects were merged. Thanks.